config/routes.rb

```diff
@@ -2,4 +2,9 @@
   devise_for :users
   root to: 'pages#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :users, except: [ :delete ] do 
```

Comment:
We need to enable the user to delete their account, but I think it's going to be related to devise paths.